packages/react-router/modules/Route.js

```diff
@@ -2,17 +2,13 @@ import React, { PropTypes } from 'react'
 import warning from 'warning'
 import matchPath from './matchPath'
 
```

Comment:
Moved `computeMatch` cause it felt cleaner as an instance method, only had to check `context.router.location` v. `props.location` in one spot.